src/main/java/interface_adapter/club_logged_in/ClubLoggedInState.java

```diff
@@ -11,10 +11,17 @@ public class ClubLoggedInState {
     private String email = "";
 
     private String password = "";
-    private String errorMessage;
+    private String message;
 
-    private ArrayList<String> membersEmail;
-    private ArrayList<String> membersName;
+    private String descriptionTextArea = "";
+
+    // Note, a single index of these two lists correspond to one post
+    private List<String> postTitles;
+    private List<String> postBodies;
+
+    // Note, a single index of these two lists correspond to one member
+    private List<String> membersName;
+    private List<String> membersEmail;
```

Comment:
this was added in my PR earlier. Will this create conflict as my PR was merged?

Reply:
Nah it shouldn't. This showed as new lines since I had to do some shuffling around of code to solve the merge conflicts after your pull request  #61 was accepted